Assets/PatchKit Patcher/Scripts/AppData/Local/Pack1Unarchiver.cs

```diff
@@ -26,21 +27,33 @@ public class Pack1Unarchiver : IUnarchiver
         private readonly byte[] _key;
         private readonly byte[] _iv;
 
+        private readonly BytesRange _range;
+
         public event UnarchiveProgressChangedHandler UnarchiveProgressChanged;
 
         public Pack1Unarchiver(string packagePath, Pack1Meta metaData, string destinationDirPath, string key, string suffix = "")
-            : this(packagePath, metaData, destinationDirPath, Encoding.ASCII.GetBytes(key), suffix)
+            : this(packagePath, metaData, destinationDirPath, Encoding.ASCII.GetBytes(key), suffix, new BytesRange(0, -1))
+        {
+            // do nothing
+        }
+
+        public Pack1Unarchiver(string packagePath, Pack1Meta metaData, string destinationDirPath, string key, string suffix, BytesRange range)
+            : this(packagePath, metaData, destinationDirPath, Encoding.ASCII.GetBytes(key), suffix, range)
         {
             // do nothing
         }
 
-        public Pack1Unarchiver(string packagePath, Pack1Meta metaData, string destinationDirPath, byte[] key, string suffix)
+        public Pack1Unarchiver(string packagePath, Pack1Meta metaData, string destinationDirPath, byte[] key, string suffix, BytesRange range)
```

Comment:
Need documentation what exactly `range` stands for.

Reply:
In comment form or something specific?

Reply:
A comment would be sufficient.

Reply:
Done, could you close this change request?

Reply:
Done ;)